sensor_array/analysis/sample_analysis.py

```diff
@@ -1013,3 +1010,116 @@ def composition_prediction_algorithm_new(array, henrys_data, gases, comps, spaci
             print('\tSubdividing Grid...\n')
             spacing = [value*0.5 for value in spacing]
             comps = subdivide_grid_from_array(filtered_comps, gases, spacing)
+
+
+def create_uniform_comp_list(gases, gas_limits, spacing, filename=None, imply_final_gas_range=True, imply_final_gas_spacing=False, filter_for_1=True, round_at=None):
```

Comment:
This would be a rejected commit because it fixes problems that are not "real" and could be fixed with how you are running the code.